### DISCUSSION TITLE
Dashboard and filter patterns (Office hours: 10/1/2020)

### DISCUSSION THREAD

**Discussion by myhlibovych** [twilio-labs/paste#806] [Office hours] [answered]
**The specific questions I have are...**
I'm creating Number Compliance Insights page, and it should provide dashboard and report with actionable items for dealing with phone numbers that are non-compliant or provisionally approved.
I haven't seen the standard approach for dashboards in Paste, and I'm not sure what is accepted practice for filtering.
**Stage of design:** 
Exploring
**Links to mockups or prototype:** https://twilio.invisionapp.com/share/5YYSGYMSBRA#/433008289_compliance_Report_Overview "
**Research**
https://docs.google.com/presentation/d/1H0GXQcXTEFf1_YEroKGdOXZRCN9hEi3qg4I6rUVha7g/edit#slide=id.g5ef346dde5_0_120
**Screen flow**
https://twilio.enterprise.slack.com/files/W01B7U8HY5N/F01B63MVC0P/pn-user_flow-compliance_insights.pdf

**Comment by serifluous** [comment, marked as answer]
Hey @myhlibovych !

1. When is your team planning to ship this? We'll have an alpha Table component out soon (which won't have a Sketch symbol to start) that will update the styling of tables in console. Depending on your ship date, we may also have a fully peer-reviewed component (with a Sketch symbol) out by launch, too.

2. I left some comments on the Invision, focused mostly on filtering for now. There seem to be quite a bit of interactive elements on all the screens, so it'll be helpful for you to walk us through them during office hours. We can help direct you to the right patterns.

3. Overall, I'd recommend taking a look at Messaging Insights for a recent example of a dashboard that uses mostly Paste components. It'll also be helpful for you to chat with folks who have designed dashboard products in Console, like @johannaweintraub and @JT1989.

4. I'd also recommend setting up some time with UX writer @ctecks https://paste.twilio.design/content/get-support#get-content-support. This is a pretty critical experience in Numbers where it may be easy to get overwhelmed, and we'll want to make sure customers have full understanding of what's happening.

**Comment by johannaweintraub** [comment]
@myhlibovych I set up some time next week for us to sync on this. BTW, the updates look fantastic.

**Comment by johannaweintraub** [comment]
Also, it appears that Voice insights was rebuilt recently with Paste. https://www.twilio.com/console/voice/insights/

**Comment by myhlibovych** [comment]
I have updated design significantly, and I need to review it how consistent it is with Paste.

https://twilio.invisionapp.com/share/6BZ6DEVE2VD

**Comment by myhlibovych** [comment]
The most up-to-date prototype is here:
https://twilio.invisionapp.com/share/EHZGRAJQ49F
I'd like to review if there's anything that should be changed to be compliant to the Paste guidelines (table pagination, filters, labels as well as interaction patterns).

**Reply by richbachman** [reply]
Hi @myhlibovych 

Would you mind starting a new discussion topic for these updated prototypes?

Thanks!

**Reply by myhlibovych** [reply]
That was my initial thought, but I wanted to keep the track of how the interface has changed.
I'll create new discussion right now.
